osbenchmark/workload/loader.py

```diff
@@ -997,11 +1021,22 @@ def read(self, workload_name, workload_spec_file, mapping_dir):
                 erroneous_lines = lines[ctx_start:ctx_end]
                 erroneous_lines.insert(line_idx - ctx_start + 1, "-" * (e.colno - 1) + "^ Error is here")
                 msg += " Lines containing the error:\n\n{}\n\n".format("\n".join(erroneous_lines))
-            msg += "The complete workload has been written to '{}' for diagnosis.".format(tmp.name)
+            msg += "The complete workload has been written to '{}' for diagnosis. \n\n".format(tmp.name)
+            console_message = f"Suggestion: Verify that [{workload_name}] workload has correctly formatted JSON files and " + \
+                "Jinja Templates. For Jinja2 errors, consider using a live Jinja2 parser. " + \
+                f"See common workload formatting errors:{WorkloadFileReader.COMMON_WORKLOAD_FORMAT_ERRORS}"
+            msg += console_message
             raise WorkloadSyntaxError(msg)
+
         except Exception as e:
+            # TypeErrors get logged here
             self.logger.exception("Could not load [%s].", workload_spec_file)
-            msg = "Could not load '{}'. The complete workload has been written to '{}' for diagnosis.".format(workload_spec_file, tmp.name)
+            msg = "Could not load '{}'. The complete workload has been written to '{}' for diagnosis. \n\n".format(
+                workload_spec_file, tmp.name)
+            console_message = f"Suggestion: Verify that [{workload_name}] workload has correctly formatted JSON files and " + \
+                "Jinja Templates. For Jinja2 errors, consider using a live Jinja2 parser. " + \
+                f"See common workload formatting errors:{WorkloadFileReader.COMMON_WORKLOAD_FORMAT_ERRORS}"
+            msg += console_message
```

Comment:
Might be helpful to consolidate the repetition into a function, so that possible future changes don't need to made in two places.